addons/binding/org.openhab.binding.regoheatpump/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Missing newline at the end of the file and can you please add the about.html to this file.